framework-plugins/lisk-framework-chain-connector-plugin/src/types.ts

```diff
@@ -33,4 +33,73 @@ export interface ChainConnectorInfo {
 	blockHeaders: chain.BlockHeader[];
 	aggregateCommits: AggregateCommit[];
 	validatorsHashPreimage: ValidatorsData[];
+	crossChainMessages: CCMsg[];
+}
+
+export interface BFTParameters {
```

Comment:
I think some or more of the types can be exported from the framework instead of creating here.